lib/view_component/slot_v2.rb

```diff
@@ -23,6 +23,9 @@ def initialize(parent)
     # the slot and return it.
     def to_s
       view_context = @parent.send(:view_context)
+
+      before_render
```

Comment:
```suggestion
```

Can we delete this line and see if tests still pass? Since SlotsV2 delegates to components and users aren't subclassing `SlotV2` I think we already have support for `before_render` in v2.

Reply:
That is interesting! I wonder if this PR has any use then (other than the added tests),